systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -191,6 +197,141 @@ void testTieredStorageWithAivenPlugin() {
+            @Step(value = "Wait until the remote folder is deleted.", expected = "The data in the remote folder are deleted.")
+
+        },
+        labels = {
+            @Label(value = TestDocsLabels.KAFKA)
+        }
+    )
+    void testTieredStorageWithAivenFileSystemPlugin() {
+        final TestStorage testStorage = new TestStorage(ResourceManager.getTestContext());
+
+        resourceManager.createResourceWithWait(
+            KafkaNodePoolTemplates.mixedPoolPersistentStorage(suiteStorage.getNamespaceName(), testStorage.getBrokerPoolName(), testStorage.getClusterName(), 1)
+                .editSpec()
+                    .withNewPersistentClaimStorage()
+                        .withSize("10Gi")
+                        .withDeleteClaim(true)
+                    .endPersistentClaimStorage()
+                .endSpec()
+                .build());
+
+        resourceManager.createResourceWithWait(KafkaTemplates.kafka(suiteStorage.getNamespaceName(), testStorage.getClusterName(), 1)
+            .editSpec()
+                .editKafka()
+                    .withImage(Environment.getImageOutputRegistry(suiteStorage.getNamespaceName(), IMAGE_NAME, BUILT_IMAGE_TAG))
+                    .withNewTieredStorageCustomTiered()
+                        .withNewRemoteStorageManager()
+                            .withClassName("io.aiven.kafka.tieredstorage.RemoteStorageManager")
+                            .withClassPath("/opt/kafka/plugins/tiered-storage/*")
+                            .addToConfig("storage.backend.class", "io.aiven.kafka.tieredstorage.storage.filesystem.FileSystemStorage")
+                            .addToConfig("storage.root", "/tmp")
+                            .addToConfig("chunk.size", "4194304")
+                        .endRemoteStorageManager()
+                    .endTieredStorageCustomTiered()
+                    // reduce the interval to speed up the test
+                    .addToConfig("remote.log.manager.task.interval.ms", 5000)
+                    .addToConfig("rlmm.config.remote.log.metadata.topic.replication.factor", 1)
+                    .addToConfig("log.retention.check.interval.ms", 5000)
+                .endKafka()
+            .endSpec()
+            .build());
+
+        resourceManager.createResourceWithWait(KafkaTopicTemplates.topic(suiteStorage.getNamespaceName(), testStorage.getTopicName(), testStorage.getClusterName())
+            .editSpec()
+                .addToConfig("file.delete.delay.ms", 1000)
+                .addToConfig("local.retention.ms", 1000)
+                // Allow tiered storage sync
+                .addToConfig("remote.storage.enable", true)
+                // Bytes retention set to 1024mb
+                .addToConfig("retention.bytes", 1073741824)
+                .addToConfig("retention.ms", 86400000)
+                // Segment size is set to 10mb to make it quicker to sync data to Minio
```

Comment:
```suggestion
                // Segment size is set to 10mb to make it quicker to sync data to remote storage
```
AFAIU Minio is not used within this test so it was just copy/paste from previous test? Or is my understanding wrong?